ts/store/migrations/addMessagesIdsByServiceId.ts

```diff
@@ -15,31 +9,10 @@ import { PersistedGlobalState } from "../reducers/types";
 export const addMessagesIdsByServiceId = (
   state: PersistedGlobalState
 ): PersistedGlobalState => {
-  const messageStatesById: MessageStateById = messagesStateByIdSelector(state);
-
-  const messagesIdsByServiceId: MessagesIdsByServiceId = Object.keys(
-    messageStatesById
-  ).reduce<MessagesIdsByServiceId>((accumulator, messageId) => {
-    const messageState = messageStatesById[messageId];
-
-    if (messageState !== undefined && pot.isSome(messageState.message)) {
-      const serviceId = messageState.message.value.sender_service_id;
-      const messagesIds = accumulator[serviceId];
-      const newMessagesIds =
-        messagesIds === undefined ? [messageId] : messagesIds.concat(messageId);
-      return {
-        ...accumulator,
-        [serviceId]: newMessagesIds
-      };
-    }
-
-    return accumulator;
-  }, {});
-
   const sectionState = {
     entities: {
       messages: {
-        idsByServiceId: messagesIdsByServiceId
+        idsByServiceId: {} // this has been removed after moving to paginated messages.
```

Comment:
Can't we remove this?

Reply:
I think so, actually this piece of code is used by a [store migration](https://github.com/pagopa/io-app/blob/5813700e665890f22808d09e7b35810940b468fa/ts/boot/configureStoreAndPersistor.ts#L80). I changed this here to leave the migration untouched.